pkgs/top-level/python-packages.nix

```diff
@@ -173,6 +173,15 @@ in {
     };
   };
 
+  base58 = buildPythonPackage rec {
```

Comment:
can you store new python packages in `./pkgs/development/python-modules` and only reference them here? This is the new policy as the file has grown too big.